setup.py

```diff
@@ -1,11 +1,9 @@
 from __future__ import absolute_import
 from __future__ import print_function
-from __future__ import unicode_literals
```

Comment:
I bet you could get away including this line without making any of the other changes. (I doubt python2 is going to care if you promote `"jsonobject/api"` from `bytes` to `unicode`

Reply:
Actually, py2 does care:
https://travis-ci.org/dimagi/jsonobject/builds/409818943